static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -64,6 +64,19 @@ const setHighMerchSlotTargeting = (slot, slotTarget): Promise<any> => {
     });
 };
 
+const adomikClassify = (): string => {
+    const rand = Math.random();
+
+    switch (true) {
+        case rand < 0.09:
```

Comment:
Ha I know! But I thought the pattern was kinda nice and figured I'd only deviate from the code they provided when it was absolutely mad, what I've got there looks sufficiently sane :)